CHANGELOG.md

```diff
@@ -92,6 +92,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Added request and response schemas for `/_cluster/decommission/awareness` ([#524](https://github.com/opensearch-project/opensearch-api-specification/pull/524))
 - Added `Bytes` component of type number ([#552](https://github.com/opensearch-project/opensearch-api-specification/pull/552))
 - Added `xy_shape` query ([#531](https://github.com/opensearch-project/opensearch-api-specification/pull/531))
+- Added `/_plugins/_flow_framework/_search` ,`state/_search`,`_provision`,`_deprovision`,`_steps`,`_status`([#508](https://github.com/opensearch-project/opensearch-api-specification/issues/508))
```

Comment:
Probably want to separate the `_search` from the first one, so it fits along with the rest.